deploy/charts/istio-csr/templates/deployment.yaml

```diff
@@ -45,7 +45,6 @@ spec:
             path: {{ .Values.app.readinessProbe.path }}
           initialDelaySeconds: 3
           periodSeconds: 7
-        command: ["cert-manager-istio-csr"]
```

Comment:
question: Why this change? Does the entrypoint change in the built container?

Reply:
Yes, I think the entrypoint changes.